.github/workflows/build-and-push.yml

```diff
@@ -4,8 +4,9 @@ on:
   push:
     branches:
       - main
+      - add-resalloc-container
 jobs:
-  build-and-push:
+  build-and-push-osh-hub:
     runs-on: ubuntu-20.04
```

Comment:
I guess github doesn't have a stream 9 option here?

Reply:
We do not self host a runner and according to the [documentation](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) Linux runners are based on Ubuntu.